src/components/content/HelpText/HelpText.stories.tsx

```diff
@@ -27,24 +27,40 @@ export default {
 
 export const Default = () => (
   <FormGroup>
-    <Label htmlFor="my-input">Input label</Label>
-    <Input id="my-input" name="my-input" />
-    <HelpText>Some help text for this input</HelpText>
+    <Label htmlFor="my-input-1">Input label</Label>
+    <Input
+      id="my-input-1"
+      name="my-input-1"
+      aria-describedby="my-input-1-helptext"
+    />
+    <HelpText id="my-input-1-helptext">Some help text for this input</HelpText>
   </FormGroup>
 );
 
 export const Positive = () => (
   <FormGroup>
-    <Label htmlFor="my-input">Input label</Label>
-    <Input id="my-input" name="my-input" />
-    <HelpText type="positive">Yay, it worked!</HelpText>
+    <Label htmlFor="my-input-2">Input label</Label>
+    <Input
+      id="my-input-2"
+      name="my-input-2"
+      aria-describedby="my-input-2-helptext"
+    />
+    <HelpText id="my-input-2-helptext" type="positive">
+      Yay, it worked!
+    </HelpText>
   </FormGroup>
 );
 
 export const Negative = () => (
   <FormGroup>
-    <Label htmlFor="my-input">Input label</Label>
-    <Input id="my-input" name="my-input" />
-    <HelpText type="negative">There was a problem</HelpText>
+    <Label htmlFor="my-input-3">Input label</Label>
+    <Input
+      id="my-input-3"
+      name="my-input-3"
+      aria-describedby="my-input-3-helptext"
```

Comment:
I think as this is an error it should have aria-invalid="true" aria-errormessage="my-input-3-helptext"

Reply:
Good point! I've added aria-invalid="true" but left it as describedby rather than errormessage because support for errormessage still seems to be a bit flaky - I can't get VoiceOver to read it out on Chrome